cypress/integration/GlossaryPopUps/GlossaryPopUps.js

```diff
@@ -0,0 +1,57 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+    // returning false here to Cypress from
+    // failing the test
+    return false;
+});
+
+
+When('user is clicking on definition for {string}', (termName) => {
+    cy.get('a.definition').first().should('be.visible');
+    cy.get('a.definition').contains(termName).click();
+    cy.wait(500);
+
+});
+
+Then('pop-up definition box is displayed for {string}', (termName) => {
+    cy.on('window:alert', (termName) => {
+        expect('div.modal__container .term').to.contains('termName');
+    })
+});
+
+And('definition box displays all {string} provided by glossary', (features) => {
+    const expText = features.split(',');
+    cy.on('window:alert', (expText) => {
+        expect('div.modal__container').to.contains('expText');
```

Comment:
So just by looking at this line of code I can tell that this step should fail - when you enclosed expText with quotes, it will look for an exact string 'expText' not a collection of expected strings